src/Session/commands.py

```diff
@@ -0,0 +1,48 @@
+class Commands:
+    def __init__(self):
+        pass
+
+    def add_food(self):
+        pass
+
+    def remove_food(self):
+        pass
+
+    def create_freezer(self):
+        pass
+
+    def create_fridge(self):
+        pass
+
+    def create_profile(self):
+        pass
+
+    def help(self):
+        pass
+
+    def list_food_storages(self):
+        pass
+
+    def list_fridges(self):
+        pass
+
+    def list_freezers(self):
+        pass
+
+    def load(self):
```

Comment:
this should take in a parameter called "profile"

Reply:
Added this in!